README.md

```
@@ -1,104 +1,22 @@
-# Silo
+<img src="https://raw.githubusercontent.com/BirknerAlex/silo/main/assets/silo.png" width="88" align="left" />
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)